nodejs/awsx/apigateway/api.ts

```diff
@@ -544,6 +604,22 @@ function addEventHandlerRouteToSwaggerSpec(
     }
 }
 
+function addRequiredParametersToSwaggerOperation(swaggerOperation: SwaggerOperation, requiredParameters: reqvalidation.Parameter[]) {
+    requiredParameters.forEach(requiredParam => {
```

Comment:
we generally don't use forEach in our codebase.  just `for (const requiredParam of requiredParameters)` is sufficient.

Reply:
For my understanding: Is this a style thing? Or is there a particular reason?

Reply:
Totally a style thing.  Since it doesn't seem to add much value, I'd prefer we just keep the code consistent with these sorts of patterns.  If you have a strong desire to use it though, go ahead :)